docs/content/docs/deployment/elastic_scaling.md

```diff
@@ -151,5 +151,45 @@ The behavior of Adaptive Scheduler is configured by [all configuration options c
+- [`jobmanager.adaptive-batch-scheduler.max-parallelism`]({{< ref "docs/deployment/config" >}}#jobmanager-adaptive-batch-scheduler-max-parallelism): The upper bound of allowed parallelism to set adaptively
+- [`jobmanager.adaptive-batch-scheduler.data-volume-per-task`]({{< ref "docs/deployment/config" >}}#jobmanager-adaptive-batch-scheduler-data-volume-per-task): The size of data volume to expect each task instance to process
+- [`jobmanager.adaptive-batch-scheduler.default-source-parallelism`]({{< ref "docs/deployment/config" >}}#jobmanager-adaptive-batch-scheduler-default-source-parallelism): The default parallelism of data source.
+
+#### Set the parallelism of operators to `-1`
+Adaptive Batch Scheduler will only decide parallelism for operators whose parallelism is not specified by users (parallelism is `-1`). So if you want the parallelism of operators to be decided automatically, you should configure as follows:
+- Set [`parallelism.default`]({{< ref "docs/deployment/config" >}}#parallelism-default) to `-1`
+- Set [`table.exec.resource.default-parallelism`]({{< ref "docs/deployment/config" >}}#table-exec-resource-default-parallelism) to `-1` in SQL jobs.
+- Don't call `setParallelism()` for operators in DataStream/DataSet jobs.
+
+### Performance tuning
+
+1. It's recommended to use [Sort Shuffle](https://flink.apache.org/2021/10/26/sort-shuffle-part1.html) and set [`taskmanager.network.memory.buffers-per-channel`]({{< ref "docs/deployment/config" >}}#taskmanager-network-memory-buffers-per-channel) to `0`. This can decouple the required network memory from parallelism, so that for large scale jobs, the "Insufficient number of network buffers" errors are less likely to happen.
+2. It's recommended to set [`jobmanager.adaptive-batch-scheduler.max-parallelism`]({{< ref "docs/deployment/config" >}}#jobmanager-adaptive-batch-scheduler-max-parallelism) to the parallelism you expect to need in the worst case. Values larger than this are not recommended, because excessive value may affect the performance. This option can affect the number of subpartitions produced by upstream tasks, large number of subpartitions may degrade the performance of hash shuffle and the performance of network transmission due to small packets.
+                                                                                                                                                                                                                                                                                      
+### Limitations
+
+- **Batch jobs only**: Adaptive Batch Scheduler only supports batch jobs.
+- **ALL-EXCHANGES-BLOCKING jobs only**: At the moment, Adaptive Batch Scheduler only supports jobs whose [shuffle mode]({{< ref "docs/deployment/config" >}}#execution-batch-shuffle-mode) is `ALL-EXCHANGES-BLOCKING`(Upstream and downstream tasks run sequentially in such jobs).
```

Comment:
> (Upstream and downstream tasks run sequentially in such jobs)

Maybe remove it, because the description of `execution-batch-shuffle-mode` already explains it in a much more detailed way.